web/docs/develop/architecture/plugins.md

```diff
@@ -24,64 +24,76 @@ static plugins. Because they are always built, we call them *builtins*.
 Tenzir offers several customization points to exchange or enhance functionality
 selectively. Here is a list of available plugin categories and plugin types:
 
-![Plugin Types](plugin-types.excalidraw.svg)
-
-### Command
-
-The command plugin adds a new command to the `tenzir` executable, at a
-configurable location in the command hierarchy. New commands can have
-sub-commands as well and allow for flexible structuring of the provided
-functionality.
-
-### Component
-
-The component plugin spawns a component inside the Tenzir node. A component is
-an [actor](actor-model) and runs in parallel with all other components.
-
-This plugin is the most generic mechanism to introduce new functionality.
-
-### Language Frontend
-
-A language frontend plugin adds an alternative parser for a query expression.
-This plugin allows for replacing the query *frontend* while using Tenzir as
-*backend* execution engine.
-
-For example, you could write a SQL plugin that takes an expression like
-`SELECT * FROM zeek.conn WHERE id.orig_h = "1.2.3.4"` and executes it on
-historical data or runs it as live query.
+```mermaid
+flowchart TB
+subgraph System
+  Command
+  click Command "#command"
+  Component
+  click Component "#component"
+end
+subgraph Storage
+  StoreActor
+  click StoreActor "#store"
+  Store
+  click Store "#store"
+end
+subgraph Format
+  Parser
+  click Parser "#parser"
+  Printer
+  click Printer "#printer"
+end
+subgraph Connector
+  Loader
+  click Loader "#loader"
+  Saver
+  click Saver "#saver"
+end
+subgraph Pipeline
+  Operator
+  click Operator "#operator"
+  AggregationFunction
+  click AggregationFunction "#aggregation-function"
+end
+```
```

Comment:
This doesn't really match the theme of the documentation. I think we should just not have this image in the first place as it doesn't add any value over the left sidebar that Docusaurus already generates from the headers.

Reply:
I think I'd be fine with removing it *iff* we add the intermediate headings that help with categorization.

Reply:
I would suggest the following:

System: 
- Component

Format:
- Parser
- Printer
- Store

Connector:
- Loader
- Saver

Pipeline:
- Operator
- Aggregation Function

Reply:
I would move Store to System or a dedicated category.

Reply:
System then, but then move it to the very bottom so the user already encountered parsers and printers.